infra/src/main/kotlin/fi/fta/geoviite/infra/tracklayout/LayoutAssetService.kt

```diff
@@ -80,8 +62,6 @@ abstract class LayoutAssetService<ObjectType : LayoutAsset<ObjectType>, DaoType
             list.filter { item -> idMatches(term, item) || contentMatches(term, item) }
         } ?: listOf()
 
-    protected open fun sortSearchResult(list: List<ObjectType>): List<ObjectType> = list
-
```

Comment:
Tämä piti jättää, mutta tässä on selkeästi joku Kotlin-juttu mikä ei käy mulle järkeen. Tästä oli tarkoitus tehdä `abstract`, mutta vaikka kuinka jumppaisin tätä signaturea, niin se haluaa olla `protected` vaikka koittaisin pakottaa sen täällä ja/tai muualla `public`:iksi. Tätä on tarkoitus pystyä käyttämään ulkoapäin, joten päädyin sitten vaan poistamaan koko abstraktin funktiomäärityksen. Jos joku tietää mikä ihme tässä on (sekä syyt tähän että potentiaaliset korjausehdotukset,) niin saa huudella.